include/xtypes/Module.hpp

```diff
@@ -286,6 +286,54 @@ class Module
         return false;
     }
 
+    EnumerationType<uint32_t>& enum_32(
```

Comment:
Why 32? What happens with the others?

Reply:
Not yet supported xD. They should be added. We may use a templated function, but internally, we must store them separately. I don't see any advantage... maybe do you have any cool idea for this, making the user's life easier as well?